### DISCUSSION TITLE
[Vote ended on 2023-08-29] Remove hpe.nimble from Ansible 10

### DISCUSSION THREAD

**Discussion by mariolenz** [ansible-community/community-topics#265] [Votes done]
`hpe.nimble` looks unmaintained (#254).

Please vote with +1 to remove it from Ansible 10 or -1 to keep it.

**Comment by mariolenz** [comment]
+1

**Comment by Andersson007** [comment]
+1

**Comment by felixfontein** [comment]
+1 - while the collection still works and is supported, it's not actively maintained.

**Comment by markuman** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by gotmax23** [comment]
+1 based on https://github.com/ansible-community/community-topics/issues/254#issuecomment-1678730501. I'll keep monitoring the situation.

**Comment by jamescassell** [comment]
+0
